frontend/styles/components/calendar/CalendarNavbar.module.scss

```
@@ -6,6 +6,13 @@
   gap: 24px;
   font-family: 'Source Sans Pro', sans-serif;
   align-self: stretch;
+  // Lets .navbarHeading/.navbarControls query *this row's* actual available
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)